javascript/stimulus_reflex.js

```diff
@@ -11,6 +11,7 @@ import {
 // A reference to the Stimulus application registered with: StimulusReflex.initialize
 //
 let stimulusApplication
+let debug = false
```

Comment:
I think this is superfluous?

Reply:
Yep. It’s an artifact and can be removed.